addons/mail/models/mail_activity.py

```diff
@@ -81,6 +81,10 @@ def _default_activity_type_for_model(self, model):
     automated = fields.Boolean(
         'Automated activity', readonly=True,
         help='Indicates this activity has been created automatically and not by any user.')
+    attachment_ids = fields.Many2many(
```

Comment:
for which activities type can we have many attachment ?
(`upload_file` upload only one attachment, and all the code I find about `action_feedback` (`markAsDone` in JS), and `action_feedback_schedule_next` only use 1 attachment)

If we use a many2one instead, the code of `get_activity_data` can also be simplified

Reply:
also, looks like `action_feedback_schedule_next` never use attachment, maybe it can be cleaned ?

Reply:
note to keep a trace: we will keep a m2m